diagnostic.md

```diff
@@ -7,36 +7,43 @@ What command do we use to do that? Use the **correct** URL from your fork of
 this repository in your answer.
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+git clone git@github.com:cathyob/git-basics-diagnostic.gi
 ```
 
 2.What do you do after cloning a repository, before you start making any
 changes/additions?
 
-<!-- Remove this comment and place your answer here. -->
+git checkout -b response
+within the file use the above in order to create and switch to a new branch
```

Comment:
While we're not specifically looking for "git checkout -b response", you are correct in that we want to create a new branch to make our changes in, rather than a master or existing branch.